config/locales/en/providers.yml

```diff
@@ -440,6 +440,32 @@ en:
         feedback_suffix: " to help us improve this service."
         view_completed_application: View completed application
         back_to_your_applications: Back to your applications
+    uploaded_evidence_collections:
+      upload_evidence:
+        list_text: 'Use this page to upload:'
+        benefit_evidence: evidence that your client receives Universal Credit
+        employment_evidence: evidence of your client's employment
```

Comment:
this should only be shown if the employed journey feature flag is enabled

Reply:
fixed below